### PR TITLE
Fix CI

### LINKED ISSUE #224: Fix releaser and setup a bot account?

### PR DESCRIPTION
**Pull request by trungleduc**
closes #224

## Code change
- Create `jupytercad` meta package
- Update version bump scripts
- Update CI scripts
- Simplify `@jupytercad/opencascade` build script
- Remove Freecad-related test assets.
- Disable `check-python` step in the check release workflow.

### CODE REVIEW

**Comment by trungleduc** [comment]
I'll update the UI test snapshot in the follow-up since the updated Galata bot script needs to be on main

File: examples/fan.stl
Comment (martinRenou) [inline examples/fan.stl:1]: This file is loaded by the Notebook example, we may want to keep it

File: packages/opencascade/src/jupytercad.opencascade.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export default null as any;
```
Comment (martinRenou) [inline packages/opencascade/src/jupytercad.opencascade.d.ts:1]: What is the purpose of this? Is this for TypeScript to copy over the js file into `lib/`?

File: ui-tests/tests/notebook.spec.ts-snapshots/dark-Notebook-ipynb-cell-2-linux.png
Comment (martinRenou) [inline ui-tests/tests/notebook.spec.ts-snapshots/dark-Notebook-ipynb-cell-2-linux.png:1]: Those reference images diff look weird. We should revert all changes in `ui-tests/tests` before merging, and fix the ui-tests in a separate PR

**Review by martinRenou — CHANGES_REQUESTED** [review]

File: packages/opencascade/src/jupytercad.opencascade.d.ts (hunk 1 above)
Comment (trungleduc) [reply packages/opencascade/src/jupytercad.opencascade.d.ts:1]: This file is there to make tsc happy and compile without buiding occ.js into 'src'. It is not copied to `lib`. The real file in `lib` is provided by webpack in the occ build process.

**Review by martinRenou — APPROVED** [review]
Thank you!

**Pull request merged**